submitted_models/costar_shafter_sensor_config_1/meshes/zr300.dae

```diff
@@ -0,0 +1,279 @@
+                </triangles>
+            </mesh>
+        </geometry>
+    </library_geometries>
+    <library_materials>
+        <material id="ID4" name="material_1">
+            <instance_effect url="#ID5" />
+        </material>
+        <material id="ID12" name="zr300_face">
+            <instance_effect url="#ID13" />
+        </material>
+        <material id="ID25" name="__auto_">
+            <instance_effect url="#ID26" />
+        </material>
+    </library_materials>
+    <library_effects>
+        <effect id="ID5">
+            <profile_COMMON>
+                <technique sid="COMMON">
+                    <lambert>
+                        <diffuse>
+                            <color>0.1725490 0.3529412 0.4705882 1.0000000</color>
+                        </diffuse>
+                    </lambert>
+                </technique>
+            </profile_COMMON>
+        </effect>
+        <effect id="ID13">
+            <profile_COMMON>
+                <newparam sid="ID15">
+                    <surface type="2D">
+                        <init_from>ID14</init_from>
+                    </surface>
+                </newparam>
+                <newparam sid="ID16">
+                    <sampler2D>
+                        <source>ID15</source>
+                    </sampler2D>
+                </newparam>
+                <technique sid="COMMON">
+                    <lambert>
+                        <diffuse>
+                            <texture texture="ID16" texcoord="UVSET0" />
+                        </diffuse>
+                        <transparent opaque="A_ONE">
+                            <texture texture="ID16" texcoord="UVSET0" />
+                        </transparent>
+                    </lambert>
+                </technique>
+            </profile_COMMON>
+        </effect>
+        <effect id="ID26">
+            <profile_COMMON>
+                <technique sid="COMMON">
+                    <lambert>
+                        <diffuse>
+                            <color>1.0000000 1.0000000 1.0000000 1.0000000</color>
+                        </diffuse>
+                    </lambert>
+                </technique>
+            </profile_COMMON>
+        </effect>
+    </library_effects>
+    <library_images>
+        <image id="ID14">
+            <init_from>zr300_face.png</init_from>
```

Comment:
After this file was moved to `materials/textures` from `meshes`, I'm seeing an error regarding finding this file.  I was able to resolve the error by changing this from `zr300_face.png` to `../materials/textures/zr300_face.png`

Reply:
updated the zr300.dae file accordingly. Thank you